blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -230,6 +202,19 @@ class LeadArt extends Component {
             autoplayPhrase={this.phrases.t('global.gallery-autoplay-button')}
             pausePhrase={this.phrases.t('global.gallery-pause-autoplay-button')}
             pageCountPhrase={(current, total) => this.phrases.t('global.gallery-page-count-text', { current, total })}
+            adElement={/* istanbul ignore next */ () => (
+              <ErrorBoundary fallback={<div>Missing Ad block</div>}>
+                <Suspense fallback={<div>Missing ad block</div>}>
```

Comment:
I updated the suspense fallback prop. The error boundary fallback should be "Missing Ad block" as when this fallback is rendered it will be because the ad-block is not available